src/tests/compiler/ast/module/sourceFileTests.ts

```diff
@@ -1405,4 +1405,79 @@ const t = 5;`;
             });
         });
     });
+
+    describe(nameof<SourceFile>(n => n.getLineAndColumnAtPos), () => {
+        function test(code: string, pos: number, expected: { line: number, column: number } | "throw") {
+            const { sourceFile } = getInfoFromText(code, { filePath: "/File.ts" });
+            if (expected === "throw") {
+                expect(() => sourceFile.getLineAndColumnAtPos(pos)).to.throw();
+            }
+            else {
+                expect(sourceFile.getLineAndColumnAtPos(pos)).to.deep.equal(expected);
+            }
+        }
+        it("should return line and column numbers at given position in single line source files", () => {
+            test(``, 0, { line: 1, column: 0 });
+            const code = `interface I { m1(): void }`;
+            test(code, 3, { line: 1, column: 3 });
+            test(code, 0, { line: 1, column: 0 });
+            test(code, code.length, { line: 1, column: code.length });
+        });
+        it("should return line and column numbers at given position in empty first line source files", () => {
+            const code = `\ninterface I { m1(): void }`;
+            test(code, 0, { line: 1, column: 0 });
+            test(code, 1, { line: 2, column: 0 });
+            test(code, code.length, { line: 2, column: code.length - 1 });
+        });
+        it("should return line and column numbers at given position in multiple lines and comments source file", () => {
+            const code = `
+/**
```

Comment:
Could you cut down this test to 2-3 lines? It's needlessly long which increases the burden when changing the code. Don't worry about testing for spaces or no spaces.